src/widget/city.c

```diff
@@ -72,7 +73,8 @@ void widget_city_draw_construction_cost(void)
     set_city_clip_rectangle();
     color_t color;
     if (cost <= city_finance_treasury()) {
-        color = COLOR_ORANGE;
+        // Color blind friendly
+        color = (scenario_property_climate() == CLIMATE_DESERT) ? COLOR_ORANGE : COLOR_ORANGE_LIGHT;
```

Comment:
By the way, do you all know that you can make commitable suggestions here?

For example:
```suggestion
        color = scenario_property_climate() == CLIMATE_DESERT ? COLOR_ORANGE : COLOR_ORANGE_LIGHT;
```

Reply:
Nice! Thanks! Saves some work.